src/Microsoft.DotNet.Interactive.Tests/Server/SerializationTests.Command_contract_has_not_been_broken.approved.AddPackage.json

```diff
@@ -1 +1 @@
-{"token":"the-token","commandType":"AddPackage","command":{"packageReference":{"packageName":"MyAwesomePackage","packageVersion":"1.2.3","isPackageVersionSpecified":true},"targetKernelName":null}}
\ No newline at end of file
+{"token":"the-token","commandType":"AddPackage","command":{"packageReference":{"packageName":"MyAwesomePackage","packageVersion":"1.2.3","isPackageVersionSpecified":true},"targetKernelName":null,"id":"command-id"}}
```

Comment:
I suspect we should hoist the `id` field up to the envelope and potentially rename it so that the intention is clearer.

Reply:
done